package.json

```diff
@@ -1,11 +1,15 @@
 {
+  "type": "module",
```

Comment:
this can be omitted on create-react-app, because under the hood CRA enables he `import` `export` syntax already